ci-operator/config/openshift/release/openshift-release-master__ocp-4.4.yaml

```diff
@@ -40,6 +40,11 @@ tests:
     - ref: baremetalds-devscripts-conf-compact
     - ref: baremetalds-devscripts-setup
     workflow: baremetalds-e2e
+- as: e2e-aws-proxy-multistep
```

Comment:
User shouldn't care about `-multistep`.  Can we drop that?  And also (possibly as a result of `make update`) remove the current hand-coded proxy entries?